class CommandError(Exception):
    """自定义异常，用于命令执行相关错误"""
    pass